### DISCUSSION TITLE
v2.12.1 update ... seeds no longer the same in a1111

### DISCUSSION THREAD

**Discussion by Stummute** [LykosAI/StabilityMatrix#928] [General]
Something changed in my SD configuration with the v2.12.1 update. none of my prior images with identical prompt and seed generate the same images anymore and other things are off. I can tell settings changed... for example an obvious one was that the path was now showing on my checkpoints dropdown and i had that turned off.

can you tell me what other configs/values were touched/changed to SD so I can fix this? I often go back on projects to pick up where i left off and they aren't generating the same images anymore as they reliably did prior to this update..

**Comment by Stummute** [comment]
I'm pretty sure, at least part of what is changing the overall seed/emphasis, is related to this. despite the option off 'sd/' is always shown... didn't do that before.

![image](https://github.com/user-attachments/assets/5286a035-a246-4f9e-82f8-73eaf7cfbad2)

Not sure that I updated to 2.12.0 before .1 ... something changed on path handling?

**Reply by mohnjiles** [reply]
That's just a folder... it has no impact on generations... we changed the Forge symlinks in 2.12.1 to include both the shared "StableDiffusion" and "Unet" folder since Forge doesn't have a separate Unet directory, and you can't symlink two directories to one.

We don't update UI settings as part of SM updates. All we did was add a subfolder. Have you tried rolling back Forge or SM to an older version to see if the issue still exists? It's possible Forge has made some changes since you last used it, or perhaps you upgraded your torch version inadvertently?

**Reply by Stummute** [reply]
thanks for reply

I tried rolling back SM by downloading 2.12.0 and putting it in it's spot, but then it wouldn't launch anything .. built in inference couldn't even see the models anymore. so i moved the settings json, stabilitymatrix.db, restarted, skipped first time setup, then tried to import the unrecognized package, but it failed to load it. 

is there another way to roll back SM so I can test that out, or is that the right process?

**Reply by mohnjiles** [reply]
You just need to replace the .exe. Did you extract it from the zip before running it?

**Reply by Stummute** [reply]
yes, and backed up/replaced the previous .exe

**Reply by Stummute** [reply]
after i tried 2.12.0 to no avail, i restored 2.12.1 and settings just to move back on with life, and i get this:

![{9887B4B2-7E50-4D03-BEFE-EFAF08869CD4}](https://github.com/user-attachments/assets/2c9165ca-8dba-42f9-ad51-b3de864584b4)

![{9B9EDE40-AB54-41F7-9307-4945FA59F810}](https://github.com/user-attachments/assets/23c3984a-a0be-4992-9d38-866ce9967a1f)

not sure why that would occur, i didnt change anything else. the models in Models\StableDiffusion were gone/deleted too... 

probably going to have to reinstall everything from scratch..
